bp/tests/BluepillTests.m

```diff
@@ -468,8 +448,11 @@ - (void)testRunUITest {
     NSError *error;
     NSString *outputDir = [BPUtils mkdtemp:[NSString stringWithFormat:@"%@/UITestsSetTempDir", tempDir] withError:&error];
     self.config.testRunnerAppPath = testRunnerPath;
+    self.config.appBundlePath = testRunnerPath;
```

Comment:
This doesn't seem right to me. Look at `newTaskWithBundle` in `BPunner.m`. I think here `testRunnerAppPath` Should be the app (testHost) not the runner, `appBundlePath` looks ok.